### PR TITLE
Adding a switch to output docker logs

### PR DESCRIPTION
**Pull request by achinthagunasekara**
##### SUMMARY

Adding a switch to output docker logs optionally if selected to do so.

```
- name: Create container if required and run command.
  docker_container:
    name: yum_reposync
    image: "{{ reposync_container }}"
    detach: no
    volumes:
      - "/tmp/bootstrap.sh:/tmp/bootstrap.sh"
    entrypoint: /bin/bash /tmp/bootstrap.sh
    log_driver: "json-file"
    output_logs: 'yes'
```
##### ISSUE TYPE

Currently we are only able to see the logs of a container if the `command`, `entrypoint` task fails.
However it's worthwhile having the ability to view these logs optionally regardless.

### CODE REVIEW

**Comment by sambanks** [comment]
This would be great!

**Comment by achinthagunasekara** [comment]
+label ready_for_review

**Comment by achinthagunasekara** [comment]
ready_for_review

File: lib/ansible/modules/cloud/docker/docker_container.py [hunk 1]
```diff
@@ -2022,6 +2034,7 @@ def main():
         oom_killer=dict(type='bool'),
         oom_score_adj=dict(type='int'),
         paused=dict(type='bool', default=False),
+        output_logs=dict(type='bool', default=False),
```
Comment (kassiansun) [inline lib/ansible/modules/cloud/docker/docker_container.py]: move this before paused to make options alphabetically ordered?

**Comment by achinthagunasekara** [comment]
@mkrizek I have tried using ready_for_review on this PR.
Is there anything I need to have this reviewed?

Thanks

**Review by kassiansun — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/cloud/docker/docker_container.py (hunk 1 above)
Comment (achinthagunasekara) [reply lib/ansible/modules/cloud/docker/docker_container.py]: @kassiansun Updated and ready_for_review again.

**Comment by achinthagunasekara** [comment]
!needs_revision

**Comment by achinthagunasekara** [comment]
ready_for_review

**Review by kassiansun — APPROVED** [review]

**Pull request merged**